index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
+</ul>
+</div>
+  
+    <button type="button" class="button-3">See Project</button>
+
+</section>
+    <!-- card4 -->
+             
+<section class="work-container3">
+    <div class="img-snapshot4">
+    <img src="images/Snapshoot4.png" alt="">
+    </div>
+    <div class="multipost1">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-3">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-3">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-3">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+    </div>
+  
+    <button type="button" class="button-4">See Project</button>
+
+</section>
+
+
+    <section class="about-wrapper">
+        <div class="header">
+            <h2>About
+                Myself</h2>
+        </div>
+        <div class="about2">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect">
+            <h3>LET’S CONNECT</h3>
+        </div>
+        <div class="socialmedia-icon">
+            <ul>
+                <li>
+                    <a href="#"><img src="images/Vectorb.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Linkedinicon.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vectorc.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vector.jpg" alt=""></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/Vectora.jpg" alt=""></a>
+                </li>
+            </ul>
+
         </div>
-    </section>
 
-     </div>
+    <button type="button" class="button-5">Get my resume</button>
+    
+    <div class="language-icons">
+        <ul>
+            <li>
+                <a href="#"><img src="images/Frame4.png" alt=""></a>
+            </li>
+        
+            <li>
+                <a href="#"><img src="images/Frame1.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Frame2.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Frame3.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Frame5.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Frame6.png" alt=""></a>
+            </li>
+        </ul>
+    </div>
```

Comment:
- [ ] Instead of using images, please consider building the elements yourself, as they can be later adjusted to be used as links or buttons. Consider using something like the following:
```
<ul>
   <li>
      Languages
      <ul>
         <li>JavaScript</li>
         <li>...
      </ul>
   </li>
   
   <li>Framworks</li>   

   <li>Skills</li>
</ul>
```

Reply:
- [ ] Nice Working on your skills section, like the previous reviewer said kindly consider not using images, for better practice and to improve the SEO.